google/cloud/bigtable/examples/table_admin_async_snippets.cc

```diff
@@ -109,6 +109,33 @@ void AsyncGetTable(cbt::TableAdmin admin, cbt::CompletionQueue cq,
   (std::move(admin), std::move(cq), argv[1]);
 }
 
+void AsyncDeleteTable(cbt::TableAdmin admin, cbt::CompletionQueue cq,
+                      std::vector<std::string> argv) {
+  if (argv.size() != 2U) {
+    throw Usage{"async-delete-table: <project-id> <instance-id> <table-id>"};
+  }
+
+  //! [async delete table]
+  namespace cbt = google::cloud::bigtable;
+  [](cbt::TableAdmin admin, cbt::CompletionQueue cq, std::string table_id) {
+    google::cloud::future<google::cloud::Status> future =
+        admin.AsyncDeleteTable(cq, table_id);
+
+    auto final =
+        future.then([table_id](google::cloud::future<google::cloud::Status> f) {
+          auto status = f.get();
+          if (!status.ok()) {
```

Comment:
s/status.ok()/status/

Reply:
I do not believe we have a `operator bool()` for `google::cloud::Status` which would be the type here, right?

Reply:
Right you are ... I was thinking of `StatusOr`.